tutorials/CMakeLists.txt

```
@@ -138,7 +138,7 @@ else()
     # The following tutorials are failing with this error:
     # IncrementalExecutor::executeFunction: symbol '__std_find_trivial_4@12' unresolved while linking [cling interface function]!
     # on Windows 32 bit and Visual Studio v17.8
-    list(APPEND roofit_veto roofit/rf509_wsinteractive.C roofit/rf614_binned_fit_problems.C)
+    list(APPEND roofit_veto roofit/rf509_wsinteractive.C roofit/rf614_binned_fit_problems.C roofit/rf615_simulation_based_inference.py)
```

```suggestion
    list(APPEND roofit_veto roofit/rf509_wsinteractive.C roofit/rf614_binned_fit_problems.C)
    # The next tutorial uses scikit-learn, which is not available on Windows 32 bit
    list(APPEND roofit_veto roofit/rf509_wsinteractive.C roofit/rf615_simulation_based_inference.py)
```

Now it looks like the comment above with the `The following tutorials are failing with this...` is also referring to you new tutorial, which is not the case. I would organize this differently.